User documents restructuring (2,3/3) (#10140)
gardener/gardener: docs/usage/shoot_basics/basic-configuration.md

---
title: Basic Shoot Configurations
weight: 6
---

> The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation the yaml structure will be mostly refered and therefore a basic understanding of it's structure will be highly usefull.

Reviewer: ```suggestion
The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation, the yaml structure will be referred to, and therefore a basic understanding of its structure will be highly useful.
```

> The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation the yaml structure will be mostly refered and therefore a basic understanding of it's structure will be highly usefull.

Reviewer 2: https://github.com/gardener/gardener/pull/9799#discussion_r1660570896

Apply all the comments posted in the old PR: https://github.com/gardener/gardener/pull/9799
